shared/versions/stack/7.6.asciidoc

```diff
@@ -0,0 +1,30 @@
+:version:                7.6.0
+////
+bare_version never includes -alpha or -beta
+////
+:bare_version:           7.6.0
+:logstash_version:       7.6.0
+:elasticsearch_version:  7.6.0
+:kibana_version:         7.6.0
+:apm_server_version:     7.6.0
+:branch:                 7.6
+:major-version:          7.x
+:prev-major-version:     6.x
+:ecs_version:            1.2
```

Comment:
I believe this should be bumped to atleast 1.3, and maybe 1.4. That would also fix the last broken link in the doc build. I'm not 100% sure though. @karenzone ??
```suggestion
:ecs_version:            1.3
```

Reply:
I bumped it to 1.4 in 7.x, expecting that 7.6 was going to be based on the info on 7.x.

I actually don't mind if it's 1.3 or 1.4, both work for me. Just picked 1.4 because it's the last published version.